src/modules/infobox/components/FeatureAreaInfo.vue

```diff
@@ -38,22 +53,36 @@ function roundValueIfGreaterThan(value, threshold, divider) {
 </script>
 
 <template>
-    <div :title="$t('area')" class="d-flex align-items-center">
+    <div :title="i18n.t('area')" class="d-flex align-items-center">
```

Comment:
When already changing this would be good to use my new tippy composable, so here you would need to use `data-tippy-content="area"` instead of `:title...` and you would need to add also an id so you can add the tippy as follow
`useTippyTooltip('#areaInfo[data-tippy-content]')`

Reply:
NOTE the selector in tippy act on the whole DOM when using the composable, therefore to avoid confusion and duplicate tippy we need to be quite restrictive here.